api/_globals/setInterval.json

```diff
@@ -149,6 +149,52 @@
             "deprecated": false
           }
         }
+      },
+      "trustedScript_param": {
+        "__compat": {
+          "description": "`trustedScript` parameter of type `TrustedScript`",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/setInterval",
```

Comment:
I can't find a spec URL for the feature in https://w3c.github.io/trusted-types/dist/spec/#toc-nav - either for this or for `setInterval`.

However if you look at the test cases listed there you can see that this can take a trusted type - https://github.com/web-platform-tests/wpt/blob/7da77c4d0c137b48ef4026db5d82e2307f4dc318/trusted-types/support/DOMWindowTimers-setTimeout-setInterval.js

Essentially before you could take code (a script) or a function callback, and now your code can also be a trusted type for a script element. I am representing that as a new parameter to the function.

1. Is this the right approach, and have I implemented correctly.
2. What do we do for spec_url in these cases?

Reply:
>  I am representing that as a new parameter to the function.

I'm not sure this is right. I don't think it is a new parameter, it's the same parameter that can now be an extra thing. Similar example, the `request` param for `fetch()`, which can be a string or a `Request`: https://developer.mozilla.org/en-US/docs/Web/API/Window/fetch#resource.

Reply:
So I would model this as "The `code` parameter can be a `TrustedScript` instance".

Reply:
Yes, with eagle-eye hindsight "of course". Fixed in https://github.com/mdn/browser-compat-data/pull/25839/commits/5a9fbb30017dd9269c053aab5f2770ee09849d9c